### DISCUSSION TITLE
github pages trailing slashes

### DISCUSSION THREAD

**Discussion by loglot** [community/community#64457] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm using default github pages and i want to have trailing slashes, for example, i would want to use "loglot.github.io" to be the landing page and "loglot.github.io/info" to be an info page, is there a way to do that? i'm not the best with code, but i'm open to learn. also I've searched quite a bit, but i didn't find anything, if you can link to a page, then do it

**Comment by mutasim77** [comment, marked as answer]
IMHO, you can achieve this by creating separate repositories for each section of your site.

**Comment by Jaber-Saed** [comment]
Absolutely, you can achieve this by utilizing GitHub Pages and some basic configuration. In your case, you want to have URLs with trailing slashes. Here's how you can set it up:

1. **Create a Repository:**
   - If you haven't already, create a GitHub repository with your username as the repository name (e.g., `loglot`). This repository will be used to host your GitHub Pages site.

2. **Enable GitHub Pages:**
   - In your repository, navigate to the "Settings" tab.
   - Scroll down to the "GitHub Pages" section.
   - Choose the "main" branch (or another branch if you prefer) as the source for GitHub Pages.

3. **Custom Domain (Optional):**
   - If you want to use a custom domain (e.g., `loglot.github.io`) instead of the default GitHub Pages domain, you can configure that under the "Custom domain" section in GitHub Pages settings.

4. **Create Directories and HTML Files:**
   - In your repository, create directories for the different pages you want (e.g., "info").
   - Inside each directory, create an `index.html` file. This file will be the default file accessed when you visit a directory.

5. **URL Structure:**
   - With this setup, you can access your pages with trailing slashes. For example:
     - Landing page: `https://loglot.github.io/`
     - Info page: `https://loglot.github.io/info/`

6. **Linking Between Pages:**
   - In your HTML files, you can create links between your pages using relative URLs. For example:
     ```html
     <!-- Link to the info page -->
     <a href="info/">Go to Info Page</a>
     ```

7. **Setting Up 404 Page (Optional):**
   - If you want to handle cases where a user enters a URL that doesn't exist, you can create a `404.html` file in your repository. GitHub Pages will automatically use this file for 404 errors.

Remember that changes might take a few minutes to propagate after you've made them.

Here's a simple example of how your repository structure might look:

```
- loglot.github.io/
  - index.html (Landing Page)
  - info/
    - index.html (Info Page)
  - 404.html (Optional)
```

With this setup, you're effectively creating a basic static website. If you're new to HTML, there are many online resources and tutorials available to help you learn the basics. You can start by learning about HTML, CSS, and how to create web pages.

As you mentioned that you're open to learning, this is a great opportunity to dive into web development basics. Once you're comfortable with these concepts, you can explore more advanced options to enhance your GitHub Pages site.

**Comment by loglot** [comment]
i did that and i just get a github pages 404

**Reply by loglot** [reply]
nevermind, i used github.com rather than github.io
